index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <div class="header">
+          <div class="bar1"> 9:41
+          <img src="/images/Group 4.png" alt="icons"></div>
+          <div class="bar2"> 
+            <h1>TRACY MIRANJA</h1>
+          <img src="/images/Icon - Menu.png" alt="togglenav">
+        </div>
+        <section class="headline">
+            <div class="banner">
+              <h1>Hey There. I’m Tracy</h1>  
+              <h2>I’am a Software Developer</h2>
+              <p>I can help you build a product , feature or website Look
+                 through some of my work and experience! If you like what 
+                 you see and have a project you need coded, don’t hestiate
+                  to contact me.</p>
+                  <div class="social-links">
+                    <a href="#"><img src="/images/Vector2.png"></a>
+                    <a href="#"><img src="/images/Vector3.png"></a>
+                    <a href="#"><img src="/images/Vector16.png"></a>
+                    <a href="#"><img src="/images/Vector5.svg"></a>
+            </div>
+            </div>
+            <div class="image">
+                <img src="/images/illustration-header-mobile.png" alt="bg">
+              </div> 
```

Comment:
- I think it will be better to render this image as a CSS background-image instead because it is just for decoration. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info. 🍒